# represents all the keypoints for human pose estimation

yolov8/KeyPoint2D[] data